workload_profiling/src/main/scala/org/apache/spark/sql/rapids/tool/profiling/ApplicationInfo.scala

```diff
@@ -0,0 +1,314 @@
+/*
+ * Copyright (c) 2021, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.rapids.tool.profiling
+
+import java.io.PrintWriter
+import java.net.URI
+
+import com.nvidia.spark.rapids.tool.profiling.ProfileArgs
+import org.apache.hadoop.conf.Configuration
+import org.apache.hadoop.fs.{FileSystem, Path}
+import org.json4s.jackson.JsonMethods.parse
+import scala.collection.Map
+import scala.collection.mutable.{ArrayBuffer, HashMap}
+import scala.io.{Codec, Source}
+
+import org.apache.spark.deploy.history.EventLogFileReader
+import org.apache.spark.internal.Logging
+import org.apache.spark.scheduler._
+import org.apache.spark.sql.{DataFrame, SparkSession}
+import org.apache.spark.ui.UIUtils
+import org.apache.spark.util._
+
+/**
+ * ApplicationInfo class saves all parsed events for future use.
+ */
+
+class ApplicationInfo(
+    val args: ProfileArgs,
+    val sparkSession: SparkSession,
+    val fileWriter: PrintWriter,
```

Comment:
similar here perhaps name pw instead of fileWriter.

Reply:
Using FileWriter instead of PrintWriter